index.html

```diff
@@ -2,10 +2,164 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <title>Title</title>
+    <meta name="description" content="IT school" />
+    <meta name="keywords" content="Front-end, HTML" />
+    <meta name="author" content="Alex Baidanov" />
+    <title>Homework 3</title>
   </head>
   <body>
-    <h1>Моя первая страница</h1>
-    <p><a href="./about" target="_blank">Об авторе</a></p>
+    <img src="./img/progg.jpg" alt="programmermeme" width="100%" />
+    <ol>
+      <li>
+        Знакомство с Web
+        <ul>
+          <li>Краткая история Web и как устроен интернет</li>
+          <li>Как устроен браузер и как он получает страницы по сети</li>
+          <li>Что такое Git</li>
+          <li>Создание первого репозитория</li>
+          <li>О файле .gitignore</li>
+        </ul>
+      </li>
+      <li>
+        HTML-вёрстка - первые шаги
+        <ul>
+          <li>Настройка рабочей среды</li>
+          <li>Краткая история HTML</li>
+          <li>Типы тегов и атрибуты</li>
+          <li>Теги для работы с текстом</li>
+          <li>Создание собственной страницы</li>
+          <li>Основы ветвления Git</li>
+          <li>Создание pull request на GitHub</li>
+        </ul>
+      </li>
+      <li>
+        Погружение в HTML-разметку
+        <ul>
+          <li>Мета-теги</li>
+          <li>Мнемоники</li>
+          <li>Теги для работы со списками</li>
+          <li>Теги для работы с таблицами</li>
+          <li>Теги для работы с формами</li>
+          <li>div и span</li>
+        </ul>
+      </li>
+    </ol>
+        <p>
```

Comment:
Good homework, but I think that here (line 47) <p> is superfluous, because you don't need to rotate the table)